### PR TITLE
Portable config without %appdata%

### PR DESCRIPTION
**Pull request by maci0**

### CODE REVIEW

File: NoFences/Services/FenceManager.cs [hunk 1]
```diff
@@ -9,10 +9,17 @@ internal class FenceManager
 
         private readonly DirectoryInfo baseDir;
         private readonly DirectoryInfo metaDir;
+        private readonly string strExeFilePath = System.Reflection.Assembly.GetExecutingAssembly().Location;
```
Comment (Twometer) [inline NoFences/Services/FenceManager.cs:12]: I think this can be a local variable in the constructor

File: NoFences/Services/FenceManager.cs [hunk 2]
```diff
@@ -9,10 +9,17 @@ internal class FenceManager
...
+
 
         public FenceManager()
         {
-            baseDir = new DirectoryInfo(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), BaseFolderName));
+            if (Directory.Exists(strExeFilePath + ".portable")){
```
Comment (Twometer) [inline NoFences/Services/FenceManager.cs:17]: Please use `Path.Combine()` for the path concatenation

**Review by Twometer — CHANGES_REQUESTED** [review]
Thanks for the cool contribution! Please see the changes I have requested, then we can merge :-)

**Review by maci0 — COMMENTED** [review]
okay. I made those changes. please take a look.

**Review by Twometer — APPROVED** [review]
Looks good to me, but is it intended that you want to merge into the 2.0 branch?

**Comment by maci0** [comment]
Yes. Locally I use a version built from the main branch with a similar patch applied. but I assume all new development is gonna happen in the 2.0 branch?

**Comment by Twometer** [comment]
Yes, the next big version will be 2.0

**Pull request merged**